src/api/spec/models/groups_user_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe GroupsUser do
+  let(:admin_user) { create(:admin_user) }
+  let(:group) { create(:group) }
+  let(:user) { create(:confirmed_user, login: 'eisendieter') }
+
+  before do
+    login(admin_user)
```

Comment:
Using `login` in model specs is weird. Better wrap the code with`admin_user.run_as do ... end`

Reply:
> Using login in model specs is weird

For sure not. We even have `ModelsAuthentication`...